src/app/api/tick/route.ts

```diff
@@ -0,0 +1,14 @@
+import dotenv from "dotenv";
+import { NextResponse } from "next/server";
+
+dotenv.config({ path: `.env.local` });
```

Comment:
**issue (performance):** Move dotenv configuration to application initialization to avoid performance overhead

Loading environment variables on every request is inefficient, especially for a frequently called endpoint. This should be done once during app startup.